libcore/src/Simulation.cpp

```diff
@@ -383,6 +332,7 @@ void Simulation::PrintStatistics(double simTime)
         }
     }
 
+    //TODO discuss deletion
```

Comment:
Should this todo be merged into master or was it a comment for review?

Reply:
Todo deleted